src/main/webapp/js/index.es6

```diff
@@ -30,6 +40,19 @@ function submissionCounter(currentDate, baseDate, submissionPerHour, baseCount)
     return formatNumber(numberOfSubmissions);
 }
 
+// looping through all the testimonials
+function loopTestimonials() {
```

Comment:
Future PR: the function name, and its comment is misleading since it only advances the current testimonial to the next one.